### DISCUSSION TITLE
Migration from Teslalogger

### DISCUSSION THREAD

**Discussion by f1x1t1x1f** [teslamate-org/teslamate#1555] [Ideas]
Is there a way to import data from Teslalogger?

**Comment by adriankumpf** [comment]
Afaik there is not enough demand for this feature. But maybe someone is willing to work on it and can contribute it. I'll move this to [💡 Ideas](https://github.com/adriankumpf/teslamate/discussions/categories/ideas).

**Reply by paulhargreaves** [reply]
I'd find this useful. I don't see anywhere documentation on what file format would be needed to be provided from an export of teslalogger (or other systems).

**Reply by kylefu8** [reply]
I think this feature is useful as well. TeslaLogger was deployed for my MY about one year ago, which recorded more than 10,000 km of driving data. But recently, it became unstable and sometimes failed to grab the data. So I decided to use Tesla Mate to replace it. But, the sad fact is that I can't migrate the historical data from TeslaLogger to Tesla Mate, which means I'm unable to check and analyze the driving data from the beginning.

I see that TeslaLogger has a tool to import data from Tesla Mate. (https://github.com/bassmaster187/TeslaLogger/blob/master/Teslamate-Import/README.md) That would be nice if Tesla Mate had a tool to migrate data from Teslalogger as well. I believe more TeslaLogger users will try to use Tesla Mate.

Thanks.

**Reply by gjuecat** [reply]
I also think this feature is useful as well.

**Reply by nsotnikov** [reply]
Yeah, the import is the only one thing is to not using tesla mate. Too much data will be lost...unfortunately.

**Reply by Trekster2000** [reply]
I would love something like this. I have about 10 months worth of data in a teslalogger database. 

I even looked at doing some kind of interim export from teslalogger into something teslamate can import, but no luck.

**Reply by juergen852** [reply]
+1
I have 2 years of teslalogger and would like to migrate to teslamate....

**Comment by CaptainDoubleYou** [comment]
I messed up my docker images on my SSD drive on Synology. I'm new to this so it seems the Teslamate folder was deleted since it installs to /var/lib. The TeslaLogger installed its container into the same folder of the docker-compose yml file. So now I have the data from TeslaLogger but I have no data in TeslaMate. I would love to be able to use TeslaMate with several months of data from TeslaLogger. Please consider helping us.
